src/utils/dates.js

```diff
@@ -10,3 +10,28 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+/**
+ * startDate and endDate are required to be two JavaScript Date objects in order for the compare function to work
+ * @param {Date} startDate
+ * @param {Date} endDate
+ */
+export function getDaysBetweenDates(startDate, endDate) {
```

Comment:
would love to see a couple more tests around the logic as we chatted in office hours (edge cases/boundaries, expected behavior, unexpected behavior)